lib/metanorma/standoc/latexml_requirement.rb

```diff
@@ -8,6 +8,8 @@ def initialize
         @minimal_version = '0.8.0'
         version_output, = Open3.capture2e("latexml --VERSION")
         @actual_version = version_output&.match(%r{\d+(.\d+)*})
+      rescue
+        warn "LaTeXML not found in PATH, please make sure that you installed LaTeXML"
```

Comment:
"LaTeXML not found in `PATH`, please ensure that LaTeXML is installed."

Reply:
Fixed